routes/articles.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+  getArticlesBySource
+} = require('../controllers/articles.controller');
+
+router.get('/articles/:source', getArticlesBySource, (req, res) => {
```

Comment:
We want `/news` to be a list of articles. We don't need to provide this type of filter.